webapp/src/pages/createApplicationForm/components/Section.js

```diff
@@ -55,90 +69,206 @@ const Section = ({
         fr: ''
       },
       required: false
-    }]});
+    }
+    const updatedSections = sections.map(s => {
+      if (s.id === input.id) {
+        s = {...input, questions: [...qsts, qst]};
+      }
+      return s;
+    });
+    setInput({...input, questions: [...qsts, qst]});
+    setApplytransition(false);
+    setQuestionAnimation(false);
   }
 
   const handleQuestions = (inpt) => {
     setInput({...input, questions: inpt});
   }
 
+  const handleOkDelete = () => {
+    const updatedSections = sections.filter(s => s.id !== input.id);
+    setSection(updatedSections);
+  }
+
+  const handleConfirm = () => {
+    setIsModelVisible(!isModelVisible);
+  }
+
+  const handleMoveSUp = () => {
+    handleMove({
+      elements: sections,
+      index: sectionIndex,
+      setState: setSection,
+      u: true
+    });
+  }
+
+  const handleMoveSDown = () => {
```

Comment:
Nit: I'm not a fan of abbreviations like this (S for Section) because it makes the code harder to read. Prefer the more explicit `handleMoveSectionDown` (same for handle[...]Q below)